setup.cfg

```diff
@@ -40,7 +40,7 @@ classifiers =
 # option for all media types, the media part may be omitted from the key.
 
 transport_can_pythoncan =
-    python-can[serial] ~= 3.3
+    python-can[serial] >= 3.3
```

Comment:
This isn't safe because there is no API compatibility guarantee between major revisions. PyUAVCAN is at the moment only compatible with PythonCAN v3.